superset/views/base.py

```diff
@@ -154,14 +157,14 @@ def wraps(self, *args, **kwargs):
     return functools.update_wrapper(wraps, f)
 
 
-def handle_api_exception(f):
+def handle_api_exception(f: Callable) -> Callable:
```

Comment:
Maybe we can be a bit more precise about `Callable` throughout? Something like `Callable[..., FlaskResponse]`?

Reply:
Across the code base `Callable` is one type which we haven't been more explicit about in terms of the return type. I wonder for simplicity whether these can remain `Callable` and in a later PR add the return type to all `Callable` types.